modules/custom/methodsdescription/main.nf

```diff
@@ -0,0 +1,50 @@
+//def VERSION=0.0.1
+
+process CUSTOM_METHODSDESCRIPTION {
+    label 'process_low'
+
+    output:
+    path "*_mqc.html"   , emit: mqc_html
+    path "versions.yml" , emit: versions
+
+    script:
+    def args = task.ext.args ?: ''
+    def doi = workflow.manifest.doi ? " (${workflow.manifest.doi})" : ""
+
+    """
+    cat <<-METHOD_DESCRIPTION > method_description_mqc.html
+    <!--
+    id: 'methods-description'
```

Comment:
To group together with other MultiQC sections generated by the pipeline. See https://github.com/nf-core/tools/pull/1389/

```suggestion
    parent_id: '${workflow.manifest.name.replace('/','-')}_run_details
    id: 'methods-description'
```